### PR TITLE
util/panic_hook, test: panics as aborts

### PR DESCRIPTION
**Pull request by overvenus**
Panics as aborts, it's helpful for debug. There is a feature `panic=abort` in rust, but unfortunately it does work in tests.

### CODE REVIEW

**Comment by overvenus** [comment]
@BusyJay It is alway disabled in tikv-server and others, it is enabled in tests only, for debug locally.

File: tests/failpoints.rs [hunk 1]
```diff
@@ -70,6 +70,10 @@ lazy_static! {
         if env::var("CI").is_ok() && env::var("LOG_FILE").is_ok() {
             self::util::init_log();
         }
+        if env::var("PANIC_ABORT").is_ok() {
+            // Panics as aborts, it's helpful for debug, but also stops tests immediately.
```
Comment (BusyJay) [inline tests/failpoints.rs]: s/debug/debugging/

**Comment by BusyJay** [comment]
LGTM

File: tests/integrations.rs [hunk 2]
```diff
@@ -65,6 +65,11 @@ fn _0_ci_setup() {
     if env::var("CI").is_ok() && env::var("LOG_FILE").is_ok() {
         self::util::init_log();
     }
+    if env::var("PANIC_ABORT").is_ok() {
+        // Panics as aborts, it's helpful for debugging,
```
Comment (siddontang) [inline tests/integrations.rs]: there are two same comments, I prefer moving these to the function for the explanation.

**Comment by overvenus** [comment]
PTAL

**Comment by hicqu** [comment]
LGTM.

**Pull request merged**